documentation/src/docs/asciidoc/release-notes/release-notes-5.6.0-M1.adoc

```diff
@@ -48,7 +48,7 @@ on GitHub.
 ==== New Features and Improvements
 
 * Documented the support of commented lines in CSV file with `@CsvFileSource`.
-
+* Add support for String delimiters in `@CsvSource`.
```

Comment:
I'm wondering whether we should also add `delimiterString` to `@CsvFileSource`.

Reply:
I was wondering about that too, I don't know if that would be useful... maybe @nrundle have some insight on this?

Reply:
It could also be done just for the sake of consistency

Reply:
I’d do it in order to be consistent.

Personally, I like being able to see the test values right by the test so I haven’t used file source.

Reply:
@junit-team/junit-lambda Any objections?

Reply:
No objections.

Reply:
@juliette-derancourt I'd say let's add it to `@CsvFileSource` for the sake of consistency then.

Reply:
+1